lagtraj/forcings/conversion/input_definitions.py

```diff
@@ -34,16 +32,18 @@
     forc_omega=[0, 1],
     forc_w=[0, 1],
     forc_geo=[0, 1],
-    nudging_u=(0, np.nan, float),
-    nudging_v=(0, np.nan, float),
-    nudging_temp=(0, np.nan, float),
-    nudging_theta=(0, np.nan, float),
-    nudging_thetal=(0, np.nan, float),
-    nudging_qv=(0, np.nan, float),
-    nudging_qt=(0, np.nan, float),
-    nudging_rv=(0, np.nan, float),
-    nudging_rt=(0, np.nan, float),
-    surfaceType=["ocean", "land", "mixed"],
+    # Allow integers only to conform to DEPHY standard, for more complex
+    # options, use the nudging "methods" below
+    nudging_u=(int),
+    nudging_v=(int),
+    nudging_temp=(int),
+    nudging_theta=(int),
+    nudging_thetal=(int),
+    nudging_qv=(int),
+    nudging_qt=(int),
+    nudging_rv=(int),
+    nudging_rt=(int),
```

Comment:
The lines above change the nudging type to integer. This is purely for agreement with the current dephy standard.